astropy/units/photometric.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+This module defines magnitude zero points and related photometric quantities.
+"""
+
+
+import numpy as _numpy
+from .core import UnitBase, def_unit
+
+from ..constants import si as _si
+from . import cgs, si, astrophys
+
+
+_ns = globals()
+
+def_unit(['Bol', 'L_bol'], _si.L_bol0, namespace=_ns, prefixes=False,
+         doc="Luminosity corresponding to absolute bolometric magnitude zero")
+def_unit(['bol', 'f_bol'], _si.L_bol0 / (4 * _numpy.pi * (10.*astrophys.pc)**2),
+         namespace=_ns, prefixes=False, doc="Irradiance corresponding to "
+         "appparent bolometric magnitude zero")
+def_unit(['ABflux'], 3631e-23 * cgs.erg * cgs.cm**-2 / si.s / si.Hz,
+         namespace=_ns, prefixes=False,
+         doc="AB magnitude zero flux density.")
```

Comment:
Given that this changed, we should add a reference to it. Indeed, arguably, it should be moved to `constants` so that we can update it...

Reply:
I'm thinking since it turns out the original definition *was* correct there's not a need to move it (rather I'll revert to that).

Reply:
Yes, fine to revert. Though now that we have the references, it would still be good to add them (or at least add a comment linking to the discussion here).